js/bybit.js

```diff
@@ -2511,6 +2503,7 @@ module.exports = class bybit extends Exchange {
         const feeCostString = this.safeString (order, 'cum_exec_fee');
         let fee = undefined;
         if (feeCostString !== undefined) {
+            const feeCurrency = market['linear'] ? market['quote'] : market['base'];
```

Comment:
Does this line hold for all market types? I doubt because spot markets are not linear/inverse.

Reply:
@kroitor That is true but for spot markets `feeCostSring` will be undefined so this line won't run, but I can change it if you want

Reply:
@carlosmiei i mean just to avoid side-effects in case it gets added to the spot response in some way.

Reply:
@kroitor should I just set it as undefined for now or check if the market is a contract before doing that line above?